### PR TITLE
Display license instead of rights 

### PR DESCRIPTION
**Pull request by kovalch**
1) for the manual corrections of the dataset: show only license 
2) as a "term of use" the licenses should be used to calculate datasets in each license category and display in the  facets

### CODE REVIEW

File: ckanext/switzerland/helpers/plugin_utils.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def ogdch_prepare_search_data_for_index(search_data):  # noqa
     search_data['res_format'] = ogdch_format_utils.prepare_formats_for_index(
         resources=validated_dict[u'resources']
     )
-    search_data['res_rights'] = [ogdch_term_utils.simplify_terms_of_use(r['rights']) for r in validated_dict[u'resources'] if 'rights' in r.keys()]  # noqa
+    search_data['res_license'] = [ogdch_term_utils.simplify_terms_of_use(r['license']) for r in validated_dict[u'resources'] if 'license' in r.keys()]  # noqa
```
Comment (bellisk) [inline ckanext/switzerland/helpers/plugin_utils.py:83]: could we break this line up please? i know we use `# noqa` on most of the long lines in this function, but this line is 159 characters long and that's twice the default line length in python, it seems a bit much!

File: ckanext/switzerland/helpers/terms_of_use_utils.py [hunk 2]
```diff
@@ -19,4 +19,4 @@ def simplify_terms_of_use(term_id):
 
     if term_id in terms:
         return term_id
-    return 'ClosedData'
+    return TERMS_OF_USE_CLOSED
```
Comment (bellisk) [inline ckanext/switzerland/helpers/terms_of_use_utils.py:22]: yay, constants! :D

File: ckanext/switzerland/logic.py [hunk 3]
```diff
@@ -194,21 +196,21 @@ def ogdch_dataset_terms_of_use(context, data_dict):
...
             if least_open is None:
-                least_open = res['rights']
+                least_open = res['license']
                 continue
-            if terms.index(res['rights']) > terms.index(least_open):
-                least_open = res['rights']
-
+            if terms.index(res['license']) > terms.index(least_open):
+                least_open = res['license']
+    log.info("HEREREEEESS")
```
Comment (bellisk) [inline ckanext/switzerland/logic.py:208]: i think you want to remove this line :)

**Review by bellisk — APPROVED** [review]

File: ckanext/switzerland/helpers/plugin_utils.py (hunk 1 above)
Comment (kovalch) [reply ckanext/switzerland/helpers/plugin_utils.py:83]: sure :)

File: ckanext/switzerland/logic.py (hunk 3 above)
Comment (kovalch) [reply ckanext/switzerland/logic.py:208]: whoops   :D

Comment (kovalch) [reply ckanext/switzerland/logic.py]: deleted

File: ckanext/switzerland/helpers/plugin_utils.py (hunk 1 above)
Comment (kovalch) [reply ckanext/switzerland/helpers/plugin_utils.py]: done

**Review by bellisk — APPROVED** [review]

**Pull request merged**